contentcuration/contentcuration/views.py

```diff
@@ -270,6 +276,67 @@ def _duplicate_node(node, sort_order=1, parent=None):
 
     return new_node
 
+
+def move_nodes(request):
+    logging.debug("Entering the move_nodes endpoint")
+
+    if request.method != 'POST':
+        raise HttpResponseBadRequest("Only POST requests are allowed on this endpoint.")
+    else:
+        data = json.loads(request.body)
+
+        try:
+            node_ids = data["node_ids"]
+            target_parent = data["target_parent"]
+            channel_id = data["channel_id"]
+        except KeyError:
+            raise ObjectDoesNotExist("Missing attribute from data: {}".format(data))
+
+        logging.info("Moving node id %s", node_ids)
+
+        nodes = [json.loads(n) for n in node_ids.split()]
+        nodes_list = []
+
+        with transaction.atomic():
+            for node_data in nodes:
+                nodes_list.append(_move_node(node_data['id'], sort_order=node_data['sort_order'], parent=target_parent, channel_id=channel_id))
+
+        nodes_serializer = ContentNodeSerializer(nodes_list, many=True)
+        return HttpResponse(json.dumps({
+            "success": True,
+            "nodes": JSONRenderer().render(nodes_serializer.data)
```

Comment:
In this direction too, it's not necessary to jsonify separately at each level; JSON (de-)serialization is recursive.